content/release-10-16-0/applications-10-16-0-bundle/10-16-0-200.md

```diff
@@ -0,0 +1,60 @@
+---
+weight: 37
+title: Release 10.16.0.200
+layout: bundle
+---
+
+<!--10.16.0.164-10.16.0.200-->
+
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Cockpit</td>
+<td>Fixed a localization issue in the tooltip of the <b>Load More</b> button in the custom Cockpit application configuration.</td>
+<td>MTM-52177</td>
+</tr>
+
+<tr>
+<td>UI</td>
+<td>Corrected Japanese localizations in the <b>App configuration</b> page and in the <b>Add smart rules</b> dialog.</td>
+<td>MTM-52050</td>
+</tr>
+
+<tr>
+<td>UI</td>
+<td>The German localization in the <b>Connect smartphone</b> wizard has been corrected.</td>
+<td>MTM-51935</td>
+</tr>
+
+<tr>
+<td>Web SDK</td>
+<td>The <code>DatapointSelectorModule</code> can now be used in a module federation plugin.</td>
+<td>MTM-52251</td>
+</tr>
+
+<tr>
+<td>Web SDK</td>
+<td>An issue with the "copy dashboard" functionality has been addressed. The device context of a configured widget is now again replaced by the context it has been pasted to. This applies, for example, to the selected asset or to datapoints.</td>

```

Comment:
not too important, but i thought about swapping the ", for example," and the preposition ("to")

Reply:
That sounds weird then for me, will leave it :-).